src/activities/PhotoHunter/PhotoHunter.js

```diff
@@ -0,0 +1,183 @@
+/* GCompris - PhotoHunter.js
+ *
+ * Copyright (C) 2016 Stefan Toncu <stefan.toncu29@gmail.com>
+ *
+ * Authors:
+ *   <Marc Le Douarain> (GTK+ version)
+ *   Stefan Toncu <stefan.toncu29@gmail.com> (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+.pragma library
+.import QtQuick 2.0 as Quick
+
+var currentLevel = 0
+var items
+var url = "qrc:/gcompris/src/activities/PhotoHunter/resource/"
+
+
+
+var dataset = [
+            {
+                "imageName": "photo1.svg",
+                "coordinates": [[12,5],[75,1170],[343,1375]],
+                "name": ["photo1_add1.svg","photo1_add2.svg","photo1_add3.svg"],
+                "radius": [0.5,0.25,1],
+                "dim": [[1.25,1.25],[0.35,0.4],[0.9,0.9]]
+            },
+            {
```

Comment:
It would be better to keep all the coordinates in a single struct like:
coordinates: [{x: 12, y:5, r: 0.5, w: 1.25, h: 1.25}, {x: 75, y:1170, r: 0.5, w: 1.25, h: 1.25}, ...]